app/static/tests/e2e/sessions.etest.ts

```diff
@@ -113,7 +113,7 @@ test.describe("Sessions tests", () => {
         await expect(await page.innerText(":nth-match(.session-label, 2)")).toBe("previous session label");
 
         // NB this will load the second load link, i.e. the older session, not the current one
-        await page.click(":nth-match(.session-load a, 2)");
+        await page.click(":nth-match(.session-load a, 2)"); // 3rd because there are two of these on the current
```

Comment:
Looks like you just added a comment but didn't change the index! (It's indexed from 1 unfortunately.)

Reply:
this is what I get for trying to edit things over ssh on the train...